src/client/components/ChannelHeadNav/ChannelHeadNav.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export default function(channelObj, urlBack) {
+  return (
+    <div>
+      <Link to={urlBack}>Back</Link>
+      <img src={channelObj.imgUrl} alt="" />
```

Comment:
I would just let them specify the channel image. No need to make it into an object. Remember the alt!!

Reply:
About the button ..I am thinking to add icon {< }instat back because it is allready a Link